presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/db/TestResourceGroupsDao.java

```diff
@@ -117,14 +117,12 @@ private static void testSelectorUpdate(H2ResourceGroupsDao dao, Map<Long, Select
                 2,
                 Optional.of(Pattern.compile("ping.*")),
                 Optional.of(Pattern.compile("ping_source")));
-        map.remove(2);
```

Comment:
I thought the `<Type>` addition was fixing the `map calls with wrong key type`. Now I see the `map.remove(2)` lines were doing the same.

Re commit message: I think a 'to' and 'the' are missing, and an object for 'remove'. So
>The fix is to remove the line whenever .remove() with wrong type is followed by .put() with the right one.